server/src/rooms/game/events/index.ts

```diff
@@ -93,6 +93,20 @@ export class BoughtAccomplishment extends GameEventWithData {
 
   apply(game: GameState): void {
     game.players[this.data.role].buyAccomplishment(this.data.accomplishment);
+    const {
+      role,
+      label,
+      science,
+      government,
+      legacy,
+      finance,
+      culture,
+      upkeep,
+      victoryPoints
+    } = this.data.accomplishment;
+    const message: string = `The ${role} purchased an accomplishment: ${label}. Science: ${science}, Government: ${government}, Legacy: ${legacy}, Finance: ${finance}, Culture: ${culture}, System Health: ${upkeep}. This added ${victoryPoints} points to their score.`;
+    const category: string = 'Bought Accomplishment';
```

Comment:
This is a minor nitpick but we should try to keep language consistent for our variable names and in user-facing text. Instead of having various synonyms for purchasing accomplishments (buyAccomplishment, boughtAccomplishment, purchaseAccomplishment) we should strive for consistency so we know what to expect API-wise on the server and the client-side instead of having to guess which one should be used.

I think we use `purchase` than `buy` in the codebase at the moment, so can we stick with `Accomplishment Purchased` or something along those lines instead? There are other places to refactor this in the server and client as well, and I'd be happy for those renames to be included in this PR as well.

```
server/src/rooms/game/events/index.ts                                           
89:export class BoughtAccomplishment extends GameEventWithData {                
98:gameEventDeserializer.register(BoughtAccomplishment);    
                                  
server/src/rooms/game/commands/index.ts                         
6:  BoughtAccomplishment,
79:      return [new BoughtAccomplishment({ accomplishment, role })];
                    
server/src/rooms/game/state/index.ts
445:  bought: Array<number>;             
454:    this.bought = new ArraySchema<Accomplishment>();       
467:    const bought = _.map(
468:      data.bought,                            
475:    this.bought.splice(0, this.bought.length, ...bought);                
482:      bought: _.map(
483:        this.bought.map(a => a.id),                       
495:  bought: ArraySchema<Accomplishment>;                                                                                         
507:      this.bought.push(new Accomplishment(accomplishment));
```

```
client/src/components/gamedashboard/bottom/events/views/AccomplishmentsSelectPurchased.vue
6:          No Bought Accomplishments. Please click 'Continue'.
9:          v-for="accomplishment in boughtAccomplishments"
66:  get boughtAccomplishments() {
67:    const bought = this.$store.getters.player.accomplishment.bought;
70:    this.purchasedAccomplishmentsLength = Object.keys(bought).length;
71:    return bought;

client/src/components/gamedashboard/left/containers/ContainerAccomplishments.vue
43:        <p v-if="boughtAccomplishments.length === 0" class="purchased-empty">
47:          v-for="accomplishment in boughtAccomplishments"
82:  get boughtAccomplishments() {
83:    return this.$store.getters.player.accomplishments.bought;

client/src/store/state.ts
40:      bought: [],

client/src/store/mutationFolder/player.ts
14:  Vue.set(p.accomplishments, 'bought', payload.data.bought);

client/src/store/mutationFolder/accomplishments.ts
18:    state.players[payload.role].accomplishments.bought.push(payload.data);
```

Reply:
resolved in newest commit (37cbda3967fc514284dc76d91ff7ff8baeee9dd6)